### PR TITLE
fixed multi app support and global node className option (#8510)

### PR DESCRIPTION
**Pull request by trailskr**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [x] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

The reason for global node className option is to provide custom class wrapper for every global nodes (teleported into body).
We use quasar components and its styles only inside out custom class (to mix it with older css worlds of bootstrap and so on...)
For more info see: #8510 

**Other information:**
The reason to mix fix and feature in one PR is to push forward feature (feature request is not reviewed within 26 days)

### CODE REVIEW

File: ui/src/install-quasar.js [hunk 1]
```diff
@@ -95,12 +88,7 @@ const installQuasar = __QUASAR_SSR_SERVER__
       })
     }
   : function (app, opts = {}) {
-    if (this.__qInstalled === true) { return }
```
Comment (iagafonov) [inline ui/src/install-quasar.js:98]: vue already checks repeated installation and worn about it

**Comment by rstoenescu** [comment]
Hi,

Thanks for this PR.

Indeed, now Vue checks if a plugin was already installed (it wasn't so a while ago). But Quasar's install flag is there for another reason too: to ensure that client-side global "watchers" / attached events are not duplicated over and over again in Quasar plugins. Regardless, this is a first good step. Merging it and tweaking it a bit then working on Quasar plugins to share their global events.

This will go into Quasar v2.0.0-beta.10.
Good work!

**Comment by rstoenescu** [comment]
@iagafonov In your particular case, do you install Quasar with different config for each Vue app?

**Comment by iagafonov** [comment]
> @iagafonov In your particular case, do you install Quasar with different config for each Vue app?

No, config is the same.

**Pull request merged**